user-lock-manager-w-keypad-support.smartapp.groovy

```diff
@@ -682,6 +711,8 @@ private initialize() {
   subscribe(theLocks, "codeReport", codereturn)
   subscribe(theLocks, "lock", codeUsed)
   subscribe(theLocks, "reportAllCodes", pollCodeReport, [filterEvents:false])
+  subscribe(location,"alarmSystemStatus",alarmStatusHandler)
+  subscribe(keypad,"codeEntered",codeEntryHandler)
```

Comment:
if (keypads) {}

works well.  Or keypad, but since you're allowing multiple keypads I'd call the variable 'keypads'.

if no keypads are set, keypads will be falsely and the block won't run.